be/src/services/entity-service.ts

```diff
@@ -3,16 +3,33 @@ import {instance} from "./persistence.service";
 import {Entity} from "../generated";
 import {Collection} from "mongodb";
 
-export default class EntityService {
+class EntityService {
     private readonly persistenceService = instance;
     private readonly entityCollection: string = "entities";
 
     private get collection(): Collection {
         return this.persistenceService.db.collection(this.entityCollection);
     }
 
-    async createEntity(vocabID: string, entity: Entity): Promise<any> {
-        return Promise.resolve(null);
+    private createNewDBO(vocabID: string, entity: Entity): Entity {
+        return {
+            id: null,
```

Comment:
@dan0bar we always have to use `_id` on DBOs, right? We do it like this on vocabs, see https://github.com/TailoredMediaProject/tm-knowledge-store/blob/main/be/src/models/dbo.models.ts#L12.

Please add your created object as interface to the `dbo.model.ts` file linked above and use it as return type.

Reply:
Yes, always refer to the `dbo.model.ts file`  file.

Reply:
@janoliver20 We will also need the conversion methods from DTO to DBO an back, I already had to do this on my branch for updating entities. To save you the typing, copy this right below the vocabDTO <-> DBO methods in `rest.router`:

```
const entityDto2Dbo = (dto: EntityDTO): Entity => ({
  _id: new ObjectId(dto.id),
  vocabulary: new ObjectId(dto.vocabulary),
  type: dto.type,
  label: dto.label,
  description: dto.description,
  created: new Date(dto.created),
  lastModified: new Date(dto.lastModified),
  externalResources: dto.externalResources,
  sameAs: dto.sameAs,
  data: dto.data
});

const entityDbo2Dto = (dbo: Entity): EntityDTO => ({
  id: dbo._id.toHexString(),
  vocabulary: dbo.vocabulary.toHexString(),
  // @ts-ignore
  type: dbo.type.toUpperCase(),
  label: dbo.label,
  description: dbo.description,
  created: dbo.created.toISOString(),
  lastModified: dbo.lastModified.toISOString(),
  externalResources: dbo.externalResources,
  sameAs: dbo.sameAs,
  data: dbo.data
});
```